apps/bfd-pipeline/bfd-pipeline-app/src/main/java/gov/cms/bfd/pipeline/app/PipelineApplication.java

```diff
@@ -148,11 +151,13 @@ ConfigLoader createConfigLoader() {
    * Runs the actual pipeline logic and throws an exception if any problems are encountered.
    *
    * @throws FatalAppException if app shutdown required
+   * @throws IOException for I/O errors
    * @throws SQLException for database errors
    */
-  private void runPipeline() throws FatalAppException, SQLException {
+  private void runPipeline() throws FatalAppException, SQLException, IOException {
 
     LOGGER.info("Application starting up!");
+    logTempDirectory();
```

Comment:
I thought it would be good to have so we can ensure the correct directory is being used.  Sensitive data is written to temp dir and we want to be sure it's in the correct volume in k8s.  @mjburling and I discussed using read-only root file system in the containers in EKS so that might make this unnecessary.  We can revisit in BFD-2916.